apps/worker/src/app/health/health.module.ts

```diff
@@ -15,12 +9,6 @@ import { SharedModule } from '../shared/shared.module';
 @Module({
   imports: [SharedModule, TerminusModule],
   controllers: [HealthController],
-  providers: [
-    DalServiceHealthIndicator,
-    QueueService,
-    QueueServiceHealthIndicator,
-    TriggerQueueService,
-    TriggerQueueServiceHealthIndicator,
```

Comment:
Moved to `SharedModule` as I needed them there also. NestJS recommends to import it as provider in just one module.